src/astarte_message_hub.rs

```diff
@@ -98,9 +98,20 @@ impl<T: AstartePublisher + AstarteSubscriber + 'static> MessageHub for AstarteMe
 
     async fn send(
         &self,
-        _request: Request<AstarteMessage>,
+        request: Request<AstarteMessage>,
     ) -> Result<Response<pbjson_types::Empty>, Status> {
-        todo!()
+        info!("Node Send Request => {:?}", request);
+
+        let astarte_message = request.into_inner();
+
+        if let Err(err) = self.astarte_handler.publish(&astarte_message).await {
+            Err(Status::internal(format!(
+                "Unable to publish astarte message, err: {:?}",
+                err
+            )))
```

Comment:
Just because it takes two lines instead of 4.
```suggestion
            let err_msg = format!("Unable to publish astarte message, err: {:?}", err);
            Err(Status::internal(err_msg))
```